frontend/src/components/Workspace/Visualize/Plot/ImagePlot.tsx

```diff
@@ -229,10 +245,23 @@ const ImagePlotChart = memo(function ImagePlotChart({
   }, [roiData])
 
   useEffect(() => {
-    onCancel()
-    onCancelAdd()
+    if (!roiFilePath || !workspaceId) return
+    dispatch(getStatus({ path: roiFilePath, workspaceId }))
+  }, [roiFilePath, workspaceId])
```

Comment:
pls add `eslint-disable-next-line`

Reply:
@ReiHashimoto I added it, pls check it again